### DISCUSSION TITLE
s

### DISCUSSION THREAD

**Discussion by krytishka** [PrismarineJS/mineflayer#2881] [Q&A]
i want those shitty spam bots to rejoin the server after being kicked
can someone help me and gimme the code but with at least one rejoinable bot?

const mineflayer = require('mineflayer')

hostip = 'perewrerwrewewrrewre.aternos.me'
porthost = '53051'

const bot1 = mineflayer.createBot({
    host: hostip,
    port: porthost,
    username: 'coolGGuy9442343424432',
    password: 'raroriro'
})

const bot2 = mineflayer.createBot({
    host: hostip,
    port: porthost,
    username: 'JuluIsCooolveryy',
    password: 'reoatasss'
})

const bot3 = mineflayer.createBot({
    host: hostip,
    port: porthost,
    username: 'tAanos222222555544',
    password: 'soocoloo444'
})

const bot4 = mineflayer.createBot({
    host: hostip,
    port: porthost,
    username: 'bebrFDFuser942555555',
    password: 'ithinkbebraiscool'
})

const bot5 = mineflayer.createBot({
    host: hostip,
    port: porthost,
    username: 'obeFbeabsdi41',
    password: 'dontyoulecturemewithurwwwwww'
})

const bot6 = mineflayer.createBot({
    host: hostip,
    port: porthost,
    username: 'AbemaFan8312944Orri',
    password: 'perdochellOO3333'
})

function whenbot2cool() {
    bot2.setControlState("jump", true)
    bot2.chat("/register reoatasss reoatasss")
    bot2.chat("/reg reoatasss reoatasss")
    bot2.chat("/login reoatasss")
    bot2.chat("RAID AHAHHAHAHAHAHAH")
    bot2.chat("UDUMB")
    bot2.chat("RAID AHAHHAHAHAHAHAH")
    bot2.chat("UDUMB")
    bot2.chat("EZ")
}
bot2.once("spawn", whenbot2cool)
bot2.once("spawn", whenbot2cool)
bot2.once("spawn", whenbot2cool)
bot2.once("spawn", whenbot2cool)
bot2.once("spawn", whenbot2cool)
bot2.once("spawn", whenbot2cool)
bot2.once("spawn", whenbot2cool)
bot2.once("spawn", whenbot2cool)

function whenbot1cool() {
    bot1.setControlState("jump", true)
    bot1.chat("/register raroriro raroriro")
    bot1.chat("/reg raroriro raroriro")
    bot1.chat("/login raroriro")
    bot1.chat("RAID AHAHHAHAHAHAHAH")
    bot1.chat("UDUMB")
    bot1.chat("RAID AHAHHAHAHAHAHAH")
    bot1.chat("UDUMB")
    bot1.chat("EZ")
}
bot1.once("spawn", whenbot1cool)
bot1.once("spawn", whenbot1cool)
bot1.once("spawn", whenbot1cool)
bot1.once("spawn", whenbot1cool)
bot1.once("spawn", whenbot1cool)
bot1.once("spawn", whenbot1cool)
bot1.once("spawn", whenbot1cool)

function whenbot3cool() {
    bot3.setControlState("jump", true)
    bot3.chat("/register soocoloo444 soocoloo444")
    bot3.chat("/reg soocoloo444 soocoloo444")
    bot3.chat("/login soocoloo444")
    bot3.chat("RAID AHAHHAHAHAHAHAH")
    bot3.chat("UDUMB")
    bot3.chat("RAID AHAHHAHAHAHAHAH")
    bot3.chat("UDUMB")
    bot3.chat("EZ")
}
bot2.once("spawn", whenbot3cool)
bot2.once("spawn", whenbot3cool)
bot2.once("spawn", whenbot3cool)
bot2.once("spawn", whenbot3cool)
bot2.once("spawn", whenbot3cool)
bot2.once("spawn", whenbot3cool)
bot2.once("spawn", whenbot3cool)
bot2.once("spawn", whenbot3cool)

function whenbot4cool() {
    bot4.setControlState("jump", true)
    bot4.chat("/register bebrFDFuser942555555 bebrFDFuser942555555")
    bot4.chat("/reg bebrFDFuser942555555 bebrFDFuser942555555")
    bot4.chat("/login bebrFDFuser942555555")
    bot4.chat("RAID AHAHHAHAHAHAHAH")
    bot4.chat("UDUMB")
    bot4.chat("RAID AHAHHAHAHAHAHAH")
    bot4.chat("UDUMB")
    bot4.chat("EZ")
}
bot4.once("spawn", whenbot4cool)
bot4.once("spawn", whenbot4cool)
bot4.once("spawn", whenbot4cool)
bot4.once("spawn", whenbot4cool)
bot4.once("spawn", whenbot4cool)
bot4.once("spawn", whenbot4cool)

function whenbot5cool() {
    bot5.setControlState("jump", true)
    bot5.chat("/register dontyoulecturemewithurwwwwww dontyoulecturemewithurwwwwww")
    bot5.chat("/reg dontyoulecturemewithurwwwwww dontyoulecturemewithurwwwwww")
    bot5.chat("/login dontyoulecturemewithurwwwwww")
    bot5.chat("RAID AHAHHAHAHAHAHAH")
    bot5.chat("UDUMB")
    bot5.chat("RAID AHAHHAHAHAHAHAH")
    bot5.chat("UDUMB")
    bot5.chat("EZ")
}

bot5.once("spawn", whenbot5cool)
bot5.once("spawn", whenbot5cool)
bot5.once("spawn", whenbot5cool)
bot5.once("spawn", whenbot5cool)
bot5.once("spawn", whenbot5cool)
bot5.once("spawn", whenbot5cool)
bot5.once("spawn", whenbot5cool)

function whenbot6cool() {
    bot6.setControlState("jump", true)
    bot6.chat("/register perdochellOO3333 perdochellOO3333")
    bot6.chat("/reg perdochellOO3333 perdochellOO3333")
    bot6.chat("/login perdochellOO3333")
    bot6.chat("RAID AHAHHAHAHAHAHAH")
    bot6.chat("UDUMB")
    bot6.chat("RAID AHAHHAHAHAHAHAH")
    bot6.chat("UDUMB")
    bot6.chat("EZ")
}

bot6.once("spawn", whenbot6cool)
bot6.once("spawn", whenbot6cool)
bot6.once("spawn", whenbot6cool)
bot6.once("spawn", whenbot6cool)
bot6.once("spawn", whenbot6cool)
bot6.once("spawn", whenbot6cool)
bot6.once("spawn", whenbot6cool)

**Comment by amoraschi** [comment]
There is an example for multiple bots
